File: Leetcode/9.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        if x < 0:
            return False
        
        else:
            m = str(x)
            temp = ''
            for k in range(len(m) - 1, -1, -1):
                temp += m[k]
            
            if temp == m:
                return True 
            
            else:
                return False

File: Leetcode/9.cpp
#include <iostream>
#include <string>

using namespace std;

class Solution {
public:
    bool isPalindrome(int x) {
        if (x < 0){
            return false;
        }
        

        else{
            string f = to_string(x);
            string Temp = "";

            for (int k=f.size()-1; k >= 0; k--) {
                Temp += f[k];
            }

            if (Temp == f){
                return true;
            }

            else{
                return false;
            }
        }
    }
};